src/components/DayPicker.jsx

```diff
@@ -301,10 +305,12 @@ export default class DayPicker extends React.Component {
         didTransitionMonth = this.maybeTransitionPrevMonth(newFocusedDate);
         break;
       case 'Home':
+        if (e) e.preventDefault();
```

Comment:
I'm curious when there will not be an `e` object in an event handler?

(it's probably best to keep this as-is, ofc, for consistency with the rest of the branches; and removing the `if` in the other branches would be semver-major, but i'd like to see us do that in the future)